tasks/app/views/people/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>People</h1>
+<div class="center">
+<% @people.each do |person| %>
+  <span>
+    <a href="/people/<%= person.id %>" >
+      <%= person.name %> - <%= person.tasks.where({:completed => nil}).length %>
```

Comment:
I would suggest moving a small bit of this code into the Person model class:
```ruby
def tasks_uncompleted
  return self.tasks.where({:completed => nil})
end
```

Then you can use `person.tasks_uncompleted.count` on this line and the one below it. (`count` is preferred to `length` for ActiveRecord model classes.)